pkgs/by-name/xv/xvidcore/package.nix

```diff
@@ -31,7 +31,10 @@ stdenv.mkDerivation rec {
     [ ]
     # Undocumented darwin hack (assembly is probably disabled due to an
     # issue with nasm, however yasm is now used)
-    ++ lib.optional stdenv.hostPlatform.isDarwin "--enable-macosx_module --disable-assembly";
+    ++ lib.optional stdenv.hostPlatform.isDarwin "--enable-macosx_module"
```

Comment:
I personally believe that using the more explicit `lib.optionals` is better, but this is opinionated.

Reply:
Thanks!